src/models/PriceDataModel.ts

```diff
@@ -27,4 +28,19 @@ export class PriceDataModel {
 
   @prop({required: false})
   fetcherSource!: string;
+
+  @prop({required: true})
```

Comment:
it is not required, it is optional only for dexes

Reply:
done